### PR TITLE
Add ControlFlowGraph class to encapsulate all information about a control flow graph.

### PR DESCRIPTION
**Pull request by AlekseyTs**

### CODE REVIEW

**Review by mavasani — APPROVED** [review]

File: src/Compilers/Core/Portable/Operations/ControlFlowGraph.cs [hunk 1]
```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+
+namespace Microsoft.CodeAnalysis.Operations
+{
+    /// <summary>
+    /// PROTOTYPE(dataflow): Add documentation
```
Comment (sharwell) [inline src/Compilers/Core/Portable/Operations/ControlFlowGraph.cs]: ❔ Do we have an analyzer capable of reporting warnings on this? If not, we could switch to the more standardized approach of using `<placeholder>` for which analyzers [are available](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1651.md).

**Review by sharwell — APPROVED** [review]

**Review by cston — APPROVED** [review]

**Review by 333fred — APPROVED** [review]
:shipit:

File: src/Compilers/Core/Portable/Operations/ControlFlowGraph.cs (hunk 1 above)
Comment (AlekseyTs) [reply src/Compilers/Core/Portable/Operations/ControlFlowGraph.cs]: @sharwell We don't need an analyzer for that. This is our standard way of marking incomplete things in feature branches. All comments like that will have to be addressed before we declare the feature complete. I am not interested in getting any diagnostics about them.

**Comment by AlekseyTs** [comment]
@dotnet/roslyn-compiler Please review, need a second sign-off

**Comment by AlekseyTs** [comment]
> need a second sign-off

I guess I got it from @333fred.

**Pull request merged**